src/test/scala/org/constellation/primitives/CheckpointBlockValidatorNelTest.scala

```diff
@@ -0,0 +1,128 @@
+package org.constellation.primitives
+
+import org.constellation.DAO
+import org.constellation.primitives.CheckpointBlockValidatorNel._
+import org.constellation.primitives.Schema.{CheckpointCacheData, Id}
+import org.constellation.primitives.storage.CheckpointService
+import org.constellation.util.HashSignature
+import org.mockito.ArgumentMatchers._
+import org.mockito.Mockito
+import org.mockito.Mockito._
+import org.scalatest.{BeforeAndAfter, FunSuite, Matchers}
+
+class CheckpointBlockValidatorNelTest extends FunSuite with Matchers with BeforeAndAfter {
```

Comment:
I think you can put there CB validator tests as well, which are currently in ValidationSpec in RandomDataTest